KsApi/MockService.swift

```diff
@@ -1244,6 +1249,14 @@
       return SignalProducer(value: GraphMutationEmptyResponseEnvelope())
     }
 
+    internal func signInWithApple(input _: SignInWithAppleInput)
+      -> SignalProducer<SignInWithAppleEnvelope, GraphError> {
+      if let error = self.signInWithAppleResult?.error {
+        return SignalProducer(error: error)
+      }
+      return SignalProducer(value: self.signInWithAppleResult?.value ?? SignInWithAppleEnvelope.template)
```

Comment:
I don't really think we should ever return defaults from these things 🤷‍♂️ but I know it's been done in a couple places.